### PR TITLE
fix: clean up from previous Momoka integration iterations

### PR DESCRIPTION
**Pull request by cesarenaldi**

### CODE REVIEW

File: packages/api-bindings/src/constants.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 import { semVer } from './SemVer';
 
-export const LENS_API_MINIMAL_SUPPORTED_VERSION = semVer('1.31.0');
+export const LENS_API_MINIMAL_SUPPORTED_VERSION = semVer('1.32.0');
```
Comment (cesarenaldi) [inline packages/api-bindings/src/constants.ts:3]: Just took the chance to update this.

File: packages/api-bindings/src/graphql/publication.graphql [hunk 2]
```diff
@@ -18,42 +18,6 @@ query GetPublication(
...
-
-    ... on Mirror {
-      ...Mirror
-    }
-
-    ... on Comment {
-      ...Comment
-    }
-  }
-}
```
Comment (cesarenaldi) [inline packages/api-bindings/src/graphql/publication.graphql:55]: Mostly deleting these are redundant with the new `GetPublication` query.

File: packages/domain/src/entities/Nft.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Challenge } from './Transactions';
+export type Challenge = string;
```
Comment (cesarenaldi) [inline packages/domain/src/entities/Nft.ts:1]: Just spotted this was positioned in the wrong file.

File: packages/react/src/transactions/infrastructure/__tests__/TransactionObserver.spec.ts [hunk 4]
```diff
@@ -87,6 +87,7 @@ describe(`Given an instance of the ${TransactionObserver.name}`, () => {
       expect(result.isFailure() && result.error.reason).toBe(TransactionErrorReason.MINING_TIMEOUT);
     }, 15_000);
 
+    // This test is temporarily skipped because the previous strategy to test stale tx does not work with the new version of Ganache
```
Comment (cesarenaldi) [inline packages/react/src/transactions/infrastructure/__tests__/TransactionObserver.spec.ts:90]: I always forget why this is skipped.

File: packages/wagmi/src/index.ts [hunk 5]
```diff
@@ -2,19 +2,23 @@ import { IBindings, RequiredSigner } from '@lens-protocol/react-web';
...
       if (chainId) {
-        try {
-          await switchNetwork({ chainId });
-        } catch (err) {
-          // best effort to switch network, if it fails, we just ignore it
-          if (!(err instanceof SwitchChainNotSupportedError)) {
-            throw err;
+        const { chain } = getNetwork();
+
+        if (chain?.id !== chainId) {
```
Comment (cesarenaldi) [inline packages/wagmi/src/index.ts:14]: Found myself in a corner... apparently `switchNetwork` hangs if asked to switch to the current network... not sure if it's ultimately caused by latest MetaMask.

**Review by krzysu — APPROVED** [review]

**Pull request merged**